src/Sandbox/restart.cpp

```diff
@@ -253,10 +261,11 @@ int main(int argc, char** argv)
   elecs[0].destroyWalkers(elecs[0].begin(), elecs[0].end());
 
   // load walkers
-  const char* restart_input = "<tmp> \
-  <mcwalkerset fileroot=\"restart\" node=\"-1\" version=\"3 0\" collected=\"yes\"/> \
+  std::string restart_input_directory = "<tmp> \
+  <mcwalkerset fileroot=\"" + directory + "restart\" node=\"-1\" version=\"3 0\" collected=\"yes\"/> \
 </tmp> \
 ";
+  const char* restart_input = restart_input_directory.c_str();
```

Comment:
No more need restart_input, just insert the right hand side to parseFromString() below.

Reply:
Yep, thanks for pointing that out